stack-orchestration/configuration/di-account-interventions-integration-admin/ais-alarm-pipeline/parameters.json

```diff
@@ -0,0 +1,90 @@
+[
+  {
+    "ParameterKey": "AllowedAccounts",
+    "ParameterValue": "324281879537"
+  },
+  {
+    "ParameterKey": "ArtifactSourceBucketArn",
+    "ParameterValue": "arn:aws:s3:::ais-alarm-pipeline-artifactpromotionbucket-9zjxdwt6itx4"
```

Comment:
What promotion bucket is this? Should be that of the AIS alarm pipeline in Staging. But in order for that bucket to be created you must enable promotion on each staging pipeline.